test/utils.js

```diff
@@ -13,11 +14,14 @@ exports.signInData = {
   password: 'holamundo2019'
 };
 
-factory.define('randomUser', User, {
+factory.define(defaultFactoryName, User, {
   name: () => faker.name.firstName(),
   last_name: () => faker.name.lastName(),
   email: () => faker.internet.email(this.name, this.last_name, 'wolox.co'),
   password: () => faker.internet.password()
 });
 
-exports.createManyUsers = size => factory.createMany('randomUser', size);
+exports.createManyUsers = size => factory.createMany(defaultFactoryName, size);
```

Comment:
what is defaultFactoryName? you never declare this variable in the function.
with default parameter I mean for example
exports.createManyUsers = (size, factoryName = 'aName') => factory.createMany(factoryName, size);